### ISSUE TITLE
Slow Download of Vortex doesn't work

### ISSUE THREAD

**Issue by CptPuddleboat** [Nexus-Mods/web-issues#1179]
## Bug/error being reported:
Can't download Vortex with the Slow Download button

## Steps to reproduce:
Attempt to download Vortex with slow download (tested in Chrome and IE, neither work)

**Comment by Pickysaurus** [comment]
I'm afraid you haven't provided much information. Are you trying to download Vortex itself or download a mod _using_ Vortex?

9 out of 10 times it will be something in your setup breaking the page. We have a help article here: https://help.nexusmods.com/article/113-troubleshooting-website-issues

**Comment by CptPuddleboat** [comment]
Sorry, I rushed it a little.

I'm running the latest updates of Windows, Chrome, and Edge. I tried to install the Vortex client in both Chrome and Edge on two different devices, and the download for the installer wouldn't work. I've tried running in incognito, disabling all of my extensions, disabling antivirus, disabling firewall, connecting to a mobile data plan, downloading from a mobile phone, all of which didn't work. I apologise if this isn't enough to go on, I was more curious whether this was a problem other people were having since I could replicate it on three separate devices (including phone).
